src/al.jl

```diff
@@ -58,7 +56,7 @@ function al(nlp :: AbstractNLPModel;
   eta = 0.5
 
   # create initial subproblem
-  al_nlp = AugLagModel(nlp, y, μ, x, cons(nlp, x))
+  al_nlp = AugLagModel(nlp, y, T(10.0), x, cons(nlp, x))
 
```

Comment:
As a parameter to the solver though, no?!

Reply:
Makes even more sense.